app/assets/javascripts/app_router.js

```diff
@@ -1,22 +1,48 @@
 var Pokedex = require("./models/pokedex.js");
 var PokedexView = require("./pokedex_view.js");
+var PokemonShowView = require("./pokemon_show_view.js");
 
-var AppRouter = function ($element) {
+var AppRouter = function (routes) {
   this.pokedex = new Pokedex();
-  this.pokedex.fetchThemAll(this.renderPokedex.bind(this));
-	this.$el = $element;
-  this.renderPokedex();
+  this.routes = routes;
+	this.container;
```

Comment:
I understand most of it. What I don't understand is the `this.container;`. What are you trying to accomplish there?